spec/support/jazz.rb

```diff
@@ -99,6 +99,14 @@ def to_graphql
     end
   end
 
+  class OverrideNameObject < GraphQL::Schema::Object
+    class << self
+      def default_graphql_name
+        "Override"
+      end
+    end
+  end
+
```

Comment:
Since this object doesn't _have_ to be part of this big example schema, how about moving it into the test file itself? For example, it could go right before the `it "..."`. 

I suggest it because I have a love-hate relationship with big example schemas like this one: on the one hand, sometimes you _need_ a complex example to demonstrate some behavior, or the easiest way to add a new test case is to extend the big example somehow. But on the other hand, this schema represents a lot of criss-crossing test changes, so over time, it becomes harder and harder to change. 

If it's not too much trouble to keep the example classes somewhere more local (somewhere closer to the test case itself), I think it'll make us happier down the road!

Reply:
Updated! That makes sense to me, only reason I added that question about my test in the PR description was because it felt weird to me in the first place. 👍